### ISSUE TITLE
vale doesn't detect .vale.ini

### ISSUE THREAD

**Issue by c0rn1ie** [jdkato/txtlint#513]
`vale ls-config` says:

```
E100 [--config] Runtime error

path '/home/user/.config/vale/vale.ini' does not exist

Execution stopped with code 1
```

I have a `.vale.ini` file in my current directory per the ~~above~~ link: https://vale.sh/docs/vale-cli/structure/#sample-repository.

_Originally posted by @c0rn1ie in https://github.com/errata-ai/vale/issues/511#issuecomment-1304401403_
